tests/pr_test.go

```diff
@@ -10,76 +10,62 @@ import (
 )
 
 // Use existing resource group
-const basicExampleDir = "examples/basic"
-const completeExampleDir = "examples/complete"
+const rootDaDir = "./"
+
+// Use existing resource group for test
+const resourceGroup = "default"
```

Comment:
remove this - see my other comment. The DA creates a new RG by default, so it not using an existing one unless you set `use_existing_resource_group` to true.